### PR TITLE
DT-1315: ✨ Use slim image with multi stage build instead

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+
+WORKDIR /app
+
+COPY package.json .
+COPY package-lock.json .
+ADD ./src/ src/
+ADD ./scripts/ scripts/
+ADD ./static/ static/
+ADD ./html-template/ html-template/
+COPY *.js ./
```
Comment (andrewrlee) [inline Dockerfile:17]: could this be managed by a .dockerignore file?

**Review by andrewrlee — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (mattops) [reply Dockerfile:17]: also, maybe could just copy everything - as the build stage gets binned at the end anyway.  I notice there is a copy everything step in the final stage also.  Maybe fine tune the `.dockerignore`  file also just to make sure we are not copying rubbish.

Comment (petergphillips) [reply Dockerfile:17]: agreed, will take a look in a future PR

**Pull request merged**